sdcm/cluster.py

```diff
@@ -1769,6 +1771,10 @@ def node_setup(node):
             node.setup_prometheus(targets=targets)
             node.install_grafana()
             node.setup_grafana()
+            # The time will be used in url of Grafana monitor,
+            # the data from this point to the end of test will
+            # be captured.
+            self.grafana_start_time = str(time.time()).split('.')[0] + '000'
```

Comment:
Let's save time.time() to `grafana_start_time`, not a non-standard string. And convert to timestamp string insider get_monitor_snapshot()